example/package.json

```diff
@@ -6,15 +6,15 @@
   "dependencies": {
     "fontfaceobserver": "^2.1.0",
     "prop-types": "^15.7.2",
-    "react": "link:../node_modules/react",
-    "react-dom": "^17.0.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
     "react-live": "^2.2.3",
     "react-scripts": "^4.0.0",
-    "react-showtime": "link:..",
+    "react-showtime": "https://github.com/azavea/react-showtime#feature/cek/netlify",
```

Comment:
I'll be changing this to `develop` prior to merging. It will be updated to point to tags as part of #4.